isolate.sh

```diff
@@ -383,46 +383,34 @@ function removeNSS(){
     title " Removing ODLM managed Namespace Scope CRs "
     msg "-----------------------------------------------------------------------"
 
-    failcheck=$(${OC} get nss --all-namespaces | grep nss-managedby-odlm || echo "failed")
+    failcheck=$(${OC} get nss ${master_ns} | grep nss-managedby-odlm || echo "failed")
```

Comment:
can probably skip this check all together, and just run `${OC} delete nss nss-managedby-odlm --ignore-not-found -n ${master_ns} || (error "unable to delete namespace scope nss-managedby-odlm in ${master_ns}")` to bypass any not found errors